certman-operator-services/certman-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 0a5e2b891727d5996421a3ca5efe91f746c75d7f 
```

Comment:
let's start with `hash: none`

That will enable staging deployment, but not prod deployment. Once we verify it works in staging, we can just replace `hash: none` with the actual hash we want to deploy.

Reply:
Fixed